src/views/ipv/page/update-details-failed.njk

```diff
@@ -20,13 +20,13 @@
               [
                    {
                    value: "delete",
-                   text: 'pages.updateDetailsFailed.content.formRadioButtons.deleteOption' | translate,
-                   hint: { text: 'pages.updateDetailsFailed.content.formRadioButtons.deleteOptionHint' | translate }
+                   text: 'pages.updateDetailsFailed.content.formRadioButtons.delete' | translate,
+                   hint: { text: 'pages.updateDetailsFailed.content.formRadioButtons.deleteHint' | translate }
                    },
                   {
-                  value: "back",
-                  text: 'pages.updateDetailsFailed.content.formRadioButtons.backOption' | translate,
-                  hint: { text: 'pages.updateDetailsFailed.content.formRadioButtons.backOptionHint' | translate }
+                  value: "returnToRp",
```

Comment:
I've just realised we're totally inconsistent about whether journey events are kebab-case or camelCase :(

The backend ones are all kebab (`not-found`, `enhanced-verification`) but the FE ones are often camel (`ukPassport`, `bankAccount`). Can we try and move towards kebab (but don't try and change all the existing ones - it'd need to be backwards compatible).

Perhaps this could be `return-to-service`?

cc @DanCorderIPV for Hilux